src/main/webapp/facebook.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
Couple of initial thoughts on the format here:

- It might be good to move these tutorial pages to a separate directory to keep them organized and away from other pages. In general, it's good practice to keep each directory to files that are all similar in some way. The more files in a directory, the harder it is to parse. So you may want to separate out directories for `webapp/tutorials/` and `webapp/views/`, or however else you want to organize things.

- The fact that `facebook.html` and `googlemeet.html` along with the existing `zoom.html` are so similar is a good hint that we should be splitting out the template (html stucture) from the data (links/text/etc). This is what a templating engine like mustache (that Rae added in an earlier PR) provides. The major benefit of doing that is it guarantees that we keep these templates in sync; otherwise if you want to make a change to one tutorial (e.g. swap the placement of the links and videos sections), you need to remember to change them all. In practice, keeping files like this completely in sync ends up being surprisingly hard to do :smile: